ngraph/python/tests/__init__.py

```diff
@@ -18,3 +18,9 @@
 # nGraph backend tests will use. It's set during pytest configuration time.
 # See `pytest_configure` hook in `conftest.py` for more details.
 BACKEND_NAME = None
+
+# test.ADDITIONAL_MODELS_DIR is a configuration variable providing the path
+# with additional ONNX models to load and test import. It's set during pytest
+# configuration time. See `pytest_configure` hook in `conftest.py` for more
+# details.
+ADDITIONAL_MODELS_DIR = None
```

Comment:
Let's use a sane default here, so we don't have to use an option. ONNX doesn't ask you to configure this, it just stores models under `~/.onnx/models`. Let's build on this and store our models under `~/.onnx/additional_models` and not allow this to be configurable. Users can override this with a `ONNX_HOME` env var.

Reply:
I can't agree with that.
Additional models dir is an option to run extra models which I got via email, im etc.
example:
I was ordered to test a huge zip with some third party models.
Its better for me to extract it in the `tmp` folder and then pass its location via option instead of searching for default dir for those.

Reply:
OK, but let's follow the ONNX library conventions for this, which is something analogous to this:
https://github.com/onnx/onnx/blob/3dbf8fe9fd051d88a75b594ca924e52e98023698/onnx/backend/test/runner/__init__.py#L204-L206

Reply:
I agree with @tsocha If we would like this to download the models there is not much thing to do here. Just add another entries here: https://github.com/openvinotoolkit/openvino/blob/master/ngraph/python/tests/test_onnx/test_zoo_models.py#L38 and that's it. 
I can set the default value for this like `${ONNX_HOME}/additional_models`.